### ISSUE TITLE
GTM id is empty

### ISSUE THREAD

**Issue by b08302359** [spatie/laravel-googletagmanager#14]
I need your help cant seem to pin point whats causing the problem I followed the steps in the documentation, seems like the id set in config was not passed or something. 

`<script async="" src="//www.google-analytics.com/analytics_debug.js"></script>`
`<script async="" src="https://www.google-analytics.com/analytics.js" style=""></script>`
`<script async="" src="//www.googletagmanager.com/gtm.js?id="></script>`
`<script>window.dataLayer = window.dataLayer || [];
dataLayer = [{"foo":"bar"}];</script>`
`<noscript>&lt;iframe src="//www.googletagmanager.com/ns.html?id="
height="0" width="0" style="display:none;visibility:hidden"&gt;&lt;/iframe&gt;</noscript>`

`<script>
(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
'//www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
})(window,document,'script','dataLayer','');
</script>`

-GET http://www.googletagmanager.com/gtm.js?id=

**Comment by freekmurze** [comment]
Have you tried running `php artisan config:clear`?

**Comment by b08302359** [comment]
Hello freekmurze,

Thanks for the reply, already tried php artisan config:clear , cache:clear and config:cache but still the same.

I just made a little changes on the GoogleTagManager.php and manually set the $this->id = "GTM-XXXXXX" it solved the empty id. 

But afterwards had an error 404 and found out that the GTM wasnt published maybe the main cause of all the problem I encountered was the unpublished GTM.
